src/types.ts

```diff
@@ -483,6 +483,23 @@ export type ManagePDNTRow = {
   interactionType?: ValidInteractionType;
 };
 
+export type UndernameMetadata = {
+  name: string;
+  targetID: string;
+  ttlSeconds: string;
+  status: number;
+  error?: string;
+  key: string;
+};
+
+export type ManageUndernameRow = {
+  attribute: string;
+  value: string;
+  editable: boolean;
+  action: any;
+  key: number;
+};
+
```

Comment:
doesn't this match an existing type? if so, make the name of that type `TableRow` or something similar

Reply:
Actually rc and antd provide types for this, i added them.

Reply:
removed